### PR TITLE
refactor: recommendations in platform adapter

### PR DESCRIPTION
**Pull request by ajperezbau**
EX-6309

### CODE REVIEW

File: packages/x-adapter-platform/src/mappers/requests/index.ts [hunk 1]
```diff
@@ -2,3 +2,4 @@ export * from './base-request.mapper';
 export * from './identifier-results-request.mapper';
 export * from './search-request.mapper';
 export * from './tagging-request.mapper';
+export * from './recommendations-request.mapper';
```
Comment (diegopf) [inline packages/x-adapter-platform/src/mappers/requests/index.ts:5]: Please, add the missing exports

File: packages/x-adapter-platform/src/types/requests/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export * from './popular-searches-request.model';
 export * from './query-suggestions-request.model';
 export * from './identifier-results-request.model';
+export * from './recommendations-request.model';
```
Comment (diegopf) [inline packages/x-adapter-platform/src/types/requests/index.ts:4]: Please, add the next-queries-request.model

File: packages/x-adapter-platform/src/types/responses/index.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
 export * from './identifier-results-response.model';
 export * from './query-suggestions-response.model';
 export * from './popular-searches-response.model';
+export * from './recommendations-response.model';
```
Comment (diegopf) [inline packages/x-adapter-platform/src/types/responses/index.ts:4]: Please, add the next-queries-response.model

**Review by diegopf — CHANGES_REQUESTED** [review]

File: packages/x-adapter-platform/src/schemas/responses/index.ts [hunk 4]
```diff
@@ -2,3 +2,4 @@ export * from './popular-searches-response.schema';
 export * from './query-suggestions-response.schema';
 export * from './next-queries-response.schema';
 export * from './identifier-results-response.schema';
+export * from './recommendations-response.schema';
```
Comment (tiborux) [inline packages/x-adapter-platform/src/schemas/responses/index.ts:5]: Add popular searches here

**Review by tiborux — CHANGES_REQUESTED** [review]
gj

**Review by diegopf — DISMISSED** [review]

**Review by mnavarroespinosa — APPROVED** [review]
looks good to me! gj!

**Review by tiborux — APPROVED** [review]

**Review by mnavarroespinosa — APPROVED** [review]

**Review by tiborux — APPROVED** [review]

**Review by diegopf — APPROVED** [review]

**Pull request merged**